client/consensus/src/collators/lookahead.rs

```diff
@@ -404,7 +403,7 @@ where
         + 'static
         + Clone,
     CIDP::InherentDataProviders: Send + InherentDataProviderExt,
-    BI: BlockImport<Block> + ParachainBlockImportMarker + Send + Sync + 'static,
+    BI: BlockImport<Block> + Send + Sync + 'static,
```

Comment:
What was the purpose of this trait bound? I wonder if there would be future issues by removing it.

Reply:
I think it was a leftover from the previous collator. If there was any issues the compilation should have detected it already

Reply:
disregard